internal/util/projutil/sdk_stamps_util.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package projutil
+
+import (
+	"regexp"
+
+	kbutil "github.com/operator-framework/operator-sdk/internal/util/kubebuilder"
+	ver "github.com/operator-framework/operator-sdk/version"
+)
+
+const (
+	OperatorSDK      = "operator-sdk"
+	BundleMediatype  = "operators.operatorframework.io.metrics.mediatype.v1"
+	BundleSDKBuilder = "operators.operatorframework.io.metrics.builder"
+	BundleSDKLayout  = "operators.operatorframework.io.metrics.project_layout"
+	sdkMediatype     = "metrics+v1"
+	OperatorBuilder  = "operators.operatorframework.io/builder"
+	OperatorLayout   = "operators.operatorframework.io/project_layout"
```

Comment:
We just put const in upper case when we need to export the value. 
Also, we create global vars when the values are used in more than one place. 

```suggestion
	OperatorBuilder  = "operators.operatorframework.io/builder"
	OperatorLayout   = "operators.operatorframework.io/project_layout"
```